yarn.lock

```diff
@@ -4290,6 +4290,15 @@
     "@octokit/types" "^6.0.3"
     universal-user-agent "^6.0.0"
 
+"@octokit/graphql@^4.6.1":
```

Comment:
so one final ask - could you update your dependency to be `^4.5.8` for this one and `yarn install` again? So that we don't have to get two versions of `@octokit/graphql`. And then later on, they will all be auto-bumped in unison by dependabot etc

Reply:
Done 👍

Reply:
Ok, should be good 👍

Reply:
Nice. Now I wish I had a "merge automatically when it goes green unless further changes are made" button.

Reply:
Oh wait, it just did, ok then :)

Reply:
😆